apps/ledger-live-mobile/src/screens/FirmwareUpdate/useUpdateFirmwareAndRestoreSettings.ts

```diff
@@ -200,17 +200,17 @@ export const useUpdateFirmwareAndRestoreSettings = ({
   // this hook controls the chaining of device actions by updating the current step
   // when needed. It basically implements a state macgine
   useEffect(() => {
-    let unrecoverableError;
+    let retriableError;
 
     switch (updateStep) {
       case "start":
         proceedToAppsBackup();
         break;
       case "appsBackup":
-        unrecoverableError =
+        retriableError =
```

Comment:
Wait but i'm not sure i understand the variable name here ahah: is it a retriable error or a none-retriable error ? Because you're checking for ex: 
```
!retriableErrors.some(err => connectManagerState.error instanceof err);
```

Reply:
it's about the manual retry, maybe retriable is not the best name in this context indeed